### PR TITLE
Improved landing.jsx mobile look

### PR DESCRIPTION
**Pull request by jinnoh**
Included a picture of mobile look below. Changed mobile responsiveness of button.jsx as well.
Screenshot 2024-04-25 185613

### CODE REVIEW

File: src/components/home/Landing.jsx [hunk 1]
```diff
@@ -8,8 +8,10 @@ const Landing = () => {
     <div className="w-full relative mb-[3%]">
       <Image src={image1} alt="Background image 1" className="w-screen" />
       <div className="flex justify-center flex-col w-1/2 h-full bg-swim-blue-200/70 top-0 absolute pl-[5%]">
-        <p className="font-bold text-white md:text-6xl mb-4">UCR Club Swim</p>
-        <p className="font-semibold text-white md:text-2xl">
+        <p className="font-bold text-white text-2xl md:text-6xl mb-4 mt-[10%] md:mt-[0%]">
```
Comment (seanquiambao) [inline src/components/home/Landing.jsx:11]:
```suggestion
        <p className="font-bold text-white text-2xl md:text-6xl mb-4 mt-[10%] md:mt-0">
```

File: src/components/home/Landing.jsx [hunk 2]
```diff
@@ -8,8 +8,10 @@ const Landing = () => {
...
+          UCR Club Swim
+        </p>
+        <p className="font-semibold text-white text-xs md:text-2xl">
           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
           eiusmod
         </p>
```
Comment (seanquiambao) [inline src/components/home/Landing.jsx:17]: image

button overflows in some screens
example : iPhone 11 Pro iOS 14.6

File: src/components/Button.jsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import Link from "next/link";
 const Button = ({ link, text, target, color = "text-black" }) => {
   return (
     <Link
-      className={` ${color} my-6 duration-300 w-fit rounded-sm md:text-2xl 2xl:text-3xl bg-swim-yellow px-8 py-2 font-bold hover:opacity-50 active:opacity-40 z-10`}
+      className={` ${color} my-3 md:my-6 duration-300 py-[1%] md:py-[2%] px-[6%] md:px-[7.5%] w-fit rounded-sm text-md md:text-2xl 2xl:text-3xl bg-swim-yellow px-8 py-2 font-bold hover:opacity-50 active:opacity-40 z-10`}
```
Comment (seanquiambao) [inline src/components/Button.jsx:6]: can you use numbers instead of percentages?  (px-1 or py-2...)

File: src/components/home/Landing.jsx (hunk 2 above)
Comment (seanquiambao) [reply src/components/home/Landing.jsx:17]: you can probably remove the top/bottom padding/margining and use gap instead.
https://tailwindcss.com/docs/gap

**Review by seanquiambao — CHANGES_REQUESTED** [review]
please also resolve merge conflicts, you can fix them through github here. (you should do this first before handling comments)

after fixing merge conflicts you can `git pull origin [this branch name]` to get the changes.

**Comment by seanquiambao** [comment]
link issues

File: src/components/Button.jsx (hunk 3 above)
Comment (jinnoh) [reply src/components/Button.jsx]: I have addressed the comments, could you review the changes!

**Review by seanquiambao — APPROVED** [review]

**Comment by seanquiambao** [comment]
resolve merge conflicts

**Pull request merged**